hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -4212,4 +4213,29 @@ private void getProcedureResult(long procId, CompletableFuture<Void> future, int
               (s, c, req, done) -> s.updateRSGroupConfig(c, req, done), resp -> null))
         ).call();
   }
+
+  private CompletableFuture<List<LogEntry>> getBalancerDecisions(
```

Comment:
Use the generic API getLogEntries() for this instead of adding one for balancer decisions.

Reply:
This is private utility method being used by generic API on L4230:
```
public CompletableFuture<List<LogEntry>> getLogEntries(LogRequest logRequest, int limit) {
```
Anything specific I can mention in method name to improve readability and the fact that this is utility method?

Reply:
MasterProtos.BalancerDecisionsRequest is not generic. This is the issue. We should be using one common generic RPC API to get slow log and balancer ring entries, and any other named ring entry type. This uses a new special master RPC for balancer decision.

Reply:
Done